src/WorkspaceDependencyViewExtension/Properties/Resources.resx

```diff
@@ -117,231 +117,246 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="DetailsMessageIncorrectVersion" xml:space="preserve">
+    <value>Some nodes in this graph were originally created with {0} v{1} which you do not have installed. Instead you have a different version of {0} installed, would you keep the installed version?</value>
```

Comment:
`Some nodes in this graph were originally created with {0} v{1}, which you do not have installed. Instead you have a different version of {0} installed. Do you wish to keep the installed version?`

Reply:
Will update